org.jacoco.core.test.validation.java5/.project

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.jacoco.core.test.java5</name>
```

Comment:
Should now read `org.jacoco.core.test.validation.java5`. Same issue with the other new `org.jacoco.core.test.validation.*` projects.

Reply:
arrrrrr

Reply:
Fixed.